src/main/java/library/common/Common.java

```diff
@@ -0,0 +1,25 @@
+package library.common;
+
+import library.BookItem;
+
+import java.util.List;
+import java.util.Random;
+
+public class Common {
```

Comment:
if you create utility class where all methods are static, it is a good practice to add private constructor

Reply:
fixed